css/styles.css

```diff
@@ -201,20 +217,21 @@
 
 .ab-column--submit-service {
     width: 342px;
-    margin-top: 56px;
+    margin-top: 20px;
+    padding-right:20px;
+	
 }
 .ab-column--submit-service p {
     padding: 5px 0;
     margin: 0;
     text-align: center;
     width: 100%;
-    background:#f6f6f6;
 }
-.ab-column--submit-service p:first-child {
-    border-top: 1px solid #f9f9f9;
+.ab-column--submit-service p:first-of-type {
+    border-top: 1px solid #e0e5e9;
```

Comment:
This is this borders original color, since `#f9f9f9` is not providing enough contrast.